validation-test/stdlib/FixedPointConversion.swift.gyb

```diff
@@ -97,6 +124,105 @@ FixedPointConversionFailure.test("${Other}To${Self}Conversion/dest=${testValue}"
 %     end # for testValue in ...
 %   end # for in all_integer_types (Other)
 % end # for in all_integer_types (Self)
+
+%   for Other, otherSignificandBits in floatNameToSignificandBits.iteritems():
```

Comment:
Yes it should be! This explains a few things about previous test failures/the lack of some failures, and I guess means gyb left `Self` hanging around even though we were out of the loop it was defined in?